mdbenchmark/analyze.py

```diff
@@ -177,19 +98,14 @@ def analyze(directory, plot, ncores, output_name):
     df.to_csv(output_name)
 
     if plot:
-        df = pd.read_csv(output_name)
+        console.warn('DEPECATED Please use \'mdbenchmark plot\' in the future')
 
-        # We only support plotting of benchmark systems from equal hosts /
-        # with equal settings
-        uniqueness = df.apply(lambda x: x.nunique())
-        if uniqueness['gromacs'] > 1 or uniqueness['host'] > 1:
-            console.error(
-                'Cannot plot benchmarks for more than one GROMACS module '
-                'and/or host.')
+        fig = Figure()
+        FigureCanvas(fig)
+        ax = fig.add_subplot(111)
 
-        # Fail if we have no values at all. This should be some edge case when
-        # a user fumbles around with the datreant categories
-        if df['gpu'].empty and df[~df['gpu']].empty:
-            console.error('There is no data to plot.')
+        df = pd.read_csv(output_name)
+        ax, lgd = plot_over_group(df, plot_cores=False, ax=ax)
```

Comment:
This does not work. You do not return `lgd` in `plot_over_group`. Instead do it like here:

https://github.com/bio-phys/MDBenchmark/blob/plotting-command/mdbenchmark/plot.py#L176-L178